package.json

```diff
@@ -40,6 +40,7 @@
     "npm-font-open-sans": "^1.1.0",
     "popper.js": "^1.14.3",
     "sw-precache": "^5.2.1",
+    "tippy.js": "^2.6.0",
```

Comment:
We're already using `vue-tippy`, is there a way we can reuse that instead of having two separate tippys included?

Reply:
Yeah, I'll wrestle with it until I figure out how it works.

Reply:
I removed the vanilla tippy.js dependency, although I'm still just using the tippy.js that's itself a dependency of VueTippy. I didn't "vue-ify" my code.